plugin/action/throttle/throttle.go

```diff
@@ -338,6 +354,10 @@ func (p *Plugin) isAllowed(event *pipeline.Event) bool {
 			throttleKey = val
 		}
 	}
+	keyLimitOverride := ""
+	if len(p.config.RedisBackendCfg.LimiterKeyField_) > 0 {
+		keyLimitOverride = event.Root.Dig(p.config.RedisBackendCfg.LimiterKeyField_...).AsString()
```

Comment:
Do we need clone the string?

```suggestion
		keyLimitOverride = strings.Clone(event.Root.Dig(p.config.RedisBackendCfg.LimiterKeyField_...).AsString())
```

Reply:
done